journal/202111.md

```diff
@@ -0,0 +1,525 @@
+
+{ ^ maybe remove the list and defer to the release notes since Pi section is pretty long already - nice problem to have }
+
+For more details check the release notes in [politeia](https://github.com/decred/politeia/releases/tag/v1.3.0) and [politeiagui](https://github.com/decred/politeiagui/releases/tag/v1.3.0) repositories, or the quick Twitter [walkthrough](https://twitter.com/lukebp/status/1462881215346880524) by @lukebp.
+
+User-facing changes merged in November:
+
+- use an unambiguous [date format](https://github.com/decred/politeiagui/pull/2640) like `08 Sep 2021`
+- fixed [Go Back](https://github.com/decred/politeiagui/pull/2658) navigation
+- ~10 other bug fixes { 2651 2661 2660 2663 2659 2665 2666 2668 2670 2673 - remove after review }
+
+Internal GUI changes:
+
+- added an extensible framework for writing [end-to-end tests](https://github.com/decred/politeiagui/pull/2624), including tools to mock API responses, generate data, and a how-to guide
+- added more end-to-end tests to prevent repeating bugs
+
+Backend changes:
+
+- [prevent](https://github.com/decred/politeia/pull/1575) RFP proposals from billing status changes (they cannot bill against the treasury, unlike RFP submissions)
+- introduced [`approved`](https://github.com/decred/politeia/pull/1578) status for proposals that cannot bill against the treasury (like RFP), to differentiate from `active` status that can bill
+- implemented [session](https://github.com/decred/politeia/pull/1564) storage in MySQL
+- implemented data integrity checks for the [`usermd`](https://github.com/decred/politeia/pull/1581) ("user metadata") plugin. This is the final check needed to proceed with importing [legacy](https://github.com/decred/politeia/issues/1425) Git proposals into the new tstore.
+- allow to set the [status](https://github.com/decred/politeia/pull/1583) of proposals created with the `pictl seedproposals` command (helps in testing)
+- added printing of execution [time statistics](https://github.com/decred/politeia/pull/1585) to the `pictl` command to help with performance testing
+- ~2 bug fixes { 1579 1561 - remove after review }
+
+`politeiavoter` command-line tool:
+
+- allow up to 100 parallel [bunches](https://github.com/decred/politeia/pull/1565)
+- reworked [help output](https://github.com/decred/politeia/pull/1580) to be more useful and match `dcrctl`. Also, only connect to the wallet if the command needs it.
+- corrected [default values](https://github.com/decred/politeia/pull/1568) in the config file and updated the sample config
+
+A notable feature of Politeia development is a formal pre-release step to test all changes in both the [backend](https://github.com/decred/politeia/issues/1573) and the [GUI](https://github.com/decred/politeiagui/issues/2653).
+
+Next release is planned to [finish](https://twitter.com/lukebp/status/1462881248205025290) the current Politeia development [proposal](https://proposals.decred.org/record/91cfcc8).
```

Comment:
This is in flux right now.  Probably best to just leave it out.  I think we're going to do a smaller release in the near future and then a final release to finish out the proposal.

Reply:
Good to know. Removed.